src/pkg/gc/gc_test.go

```diff
@@ -94,39 +120,36 @@ func generateFiveTestImages() (string, map[string]string) {
 	twelweHoursOld := timeNow.Add(-12 * time.Hour)
 	dayOld := timeNow.Add(-24 * time.Hour)
 
-	idsAndDatesMap := make(map[string]string)
-	idsAndDatesMap["8dfafdbc3a40"] = strconv.FormatInt(timeNow.Unix(), 10)
-	idsAndDatesMap["9cd87474be90"] = strconv.FormatInt(threeSecondsOld.Unix(), 10)
-	idsAndDatesMap["3176a2479c92"] = strconv.FormatInt(fiveMinutesOld.Unix(), 10)
-	idsAndDatesMap["4cb07b47f9fb"] = strconv.FormatInt(twelweHoursOld.Unix(), 10)
-	idsAndDatesMap["5c76a2479c92"] = strconv.FormatInt(dayOld.Unix(), 10)
+	idsAndDatesMap := make(map[string]int64)
+	idsAndDatesMap["8dfafdbc3a40"] = timeNow.Unix()
+	idsAndDatesMap["9cd87474be90"] = threeSecondsOld.Unix()
+	idsAndDatesMap["3176a2479c92"] = fiveMinutesOld.Unix()
+	idsAndDatesMap["4cb07b47f9fb"] = twelweHoursOld.Unix()
+	idsAndDatesMap["5c76a2479c92"] = dayOld.Unix()
 
-	var imageList []string
+	var imageList []idAndCreated
 	imageHistoryList := make(map[string]string)
 
 	for id, date := range idsAndDatesMap {
-		imageListInfo := `
-     {
-             "Id":"` + id + `",
-             "Created":` + date + `
-     }`
+		imageListInfo := idAndCreated{Id: id, Created: date}
 		imageList = append(imageList, imageListInfo)
-		imageHistory := `
-	 [{
-             "Id":"` + id + `",
-             "Created":` + date + `,
-             "Tags": [
-                "test:` + id + `"
-             ]
-	 }]`
-
-		imageHistoryList[id] = imageHistory
+
+		imageHistory := mustMarshal(idAndCreated{Id: id, Created: date})
+		imageHistoryList[id] = string(imageHistory)
 	}
 
-	imageListAsJson := strings.Join(imageList[:], ",")
-	imageListAsJson = `[` + imageListAsJson + "\n" + `  ]`
+	imageListAsJson := mustMarshal(imageList)
+
+	return string(imageListAsJson), imageHistoryList
+}
+
+func mustMarshal(data interface{}) []byte {
+	out, err := json.Marshal(data)
+	if err != nil {
+		panic(err)
```

Comment:
Will we get a bugsnag exeption or DD event that we can monitor for this? If not I think we should have one.